x-pack/test/apm_api_integration/tests/alerts/anomaly_alert.spec.ts

```diff
@@ -86,7 +86,7 @@ export default function ApiTest({ getService }: FtrProviderContext) {
               windowSize: 5,
               windowUnit: 'h',
               anomalySeverityType: ML_ANOMALY_SEVERITY.WARNING,
-              anomalyDetectorTypes: [ApmMlDetectorType.txLatency],
+              anomalyDetectorTypes: ['txLatency'],
```

Comment:
Could you please add a test to check the reason message 

similar to
[[alerts](https://github.com/elastic/kibana/blob/80bf35786209abfa0019c48cc0a7092491311b6a/x-pack/test/apm_api_integration/tests/alerts/transaction_duration.spec.ts#L152-L163)](https://github.com/elastic/kibana/blob/80bf35786209abfa0019c48cc0a7092491311b6a/x-pack/test/apm_api_integration/tests/alerts/transaction_duration.spec.ts#L152-L163)